_data/entertainment.yml

```diff
@@ -145,8 +145,8 @@ websites:
     url: https://www.smashcast.tv/
     img: smashcast.png
     tfa:
-      - software
-    doc: https://help.smashcast.tv/customer/en/portal/articles/2779887
+      - totp
+    doc: https://support.desk.com/customer/portal/articles/1894034
```

Comment:
That doc link is for Salesforce Desk. Looks like Smashcast's help site is down at the moment so let's leave it empty for now.
```suggestion
```